src/Immunisatie-0-1/_components/variable.immunization.medication.url.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<nts:component xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript">
+    <variable>
+        <name value="{$name}"/>
+        <!-- Pick the absolute url as is or create an absolute url when a relative url is given by combining the base url given in the Bundle self link with the relative reference.-->
+        <expression
+            value="iif(Bundle.entry.where(resource.is(Immunization)).resource[{$sequenceNumber}].extension.where(url = 'http://nictiz.nl/fhir/StructureDefinition/ext-Vaccination.PharmaceuticalProduct').value.reference.startsWith('Medication/'),
+            Bundle.link.where(relation='self').url.replaceMatches('Immunization[/?].*$','') + Bundle.entry.where(resource.is(Immunization)).resource[{$sequenceNumber}].extension.where(url = 'http://nictiz.nl/fhir/StructureDefinition/ext-Vaccination.PharmaceuticalProduct').value.reference,
+            Bundle.entry.where(resource.is(Immunization)).resource[{$sequenceNumber}].extension.where(url = 'http://nictiz.nl/fhir/StructureDefinition/ext-Vaccination.PharmaceuticalProduct').value.reference)"/>
+        <sourceId value="immunization-search-response"/>
```

Comment:
Ah hier. Maar dan ben ik erop tegen dat een sourceId hard gecodeerd in een component gebruikt wordt. Zou ook een parameter moeten zijn.